.buildkite/winlogbeat/pipeline.winlogbeat.yml

```diff
@@ -2,48 +2,149 @@
 name: "beats-winlogbeat"
 
 env:
-  BEATS_PROJECT_NAME: "winlogbeat"
```

Comment:
Question: are we intentionally removing `BEATS_PROJECT_NAME`? It is used in various scripts

Reply:
> Question: are we intentionally removing `BEATS_PROJECT_NAME`? It is used in various scripts

we yes want to retire it with an aim to remove as many scripts as possible in the near future.

Reply:
Yep, that was asked during previous PRs.